include/igl/copyleft/comiso/miq.h

```diff
@@ -54,24 +54,43 @@ namespace igl
       const Eigen::PlainObjectBase<DerivedV> &PD2,
       Eigen::PlainObjectBase<DerivedU> &UV,
       Eigen::PlainObjectBase<DerivedF> &FUV,
-      double scale = 30.0,
-      double stiffness = 5.0,
-      bool direct_round = false,
+      double GradientSize = 30.0,
```

Comment:
Since we're at it, I would update both the code and the doc to use camelCase (starting with lowercase) for all variables that are not matrices. @alecjacobson, @danielepanozzo we don't have a naming convention for variables in the style guidelines? (Other than they should be short.)